program/rust/src/tests/test_upd_price.rs

```diff
@@ -248,7 +272,11 @@ fn test_upd_price() {
         assert_eq!(price_data.valid_slot_, 5);
         assert_eq!(price_data.agg_.pub_slot_, 6);
         assert_eq!(price_data.agg_.price_, 81);
-        assert_eq!(price_data.agg_.status_, PC_STATUS_TRADING);
+        assert_eq!(price_data.agg_.status_, PC_STATUS_UNKNOWN);
+        assert_eq!(price_data.prev_slot_, 5);
+        assert_eq!(price_data.prev_price_, 81);
+        assert_eq!(price_data.prev_conf_, 2);
+        assert_eq!(price_data.prev_timestamp_, 0);
     }
 
     // Crank one more time and aggregate should be unknown
```

Comment:
`// Crank one more time and aggregate should be unknown` comments now are weird since we don't need to crank the aggregation

Reply:
removed the whole assert blocks for "crank again" comments